streams-contrib/streams-amazon-aws/streams-persist-s3/src/main/java/org/apache/streams/s3/S3Configurator.java

```diff
@@ -64,15 +64,15 @@ public static S3WriterConfiguration detectWriterConfiguration(Config s3) {
 
         try {
             s3Configuration = mapper.readValue(s3.root().render(ConfigRenderOptions.concise()), S3WriterConfiguration.class);
+
+            if(!s3Configuration.getWriterPath().endsWith("/")) {
+                s3Configuration.setWriterPath(s3Configuration.getWriterPath() + "/");
+            }
         } catch (Exception e) {
             e.printStackTrace();
```

Comment:
since you are in here, remove e.printStackTrace().  If the exception needs to be output to the console, it needs to be output to the logs.